src/sentry/models/options/project_template_option.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model, sane_repr
+from sentry.db.models.fields import PickledObjectField
+
+
+@region_silo_only_model
+class ProjectTemplateOption(Model):
+    """
+    A ProjectTemplateOption is a templated version of a project
+
+    It is used to store the values that are shared between different
+    projects across the organization.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    project_template = FlexibleForeignKey("sentry.ProjectTemplate", related_name="options")
+    key = models.CharField(max_length=64)
+    value = PickledObjectField()
```

Comment:
I'm not sure we should be adding more pickled data to our database. Using a text field with JSON encoded payloads would be an improvement over the current option schema.

Reply:
Currently, it's freeform. We'll be aligning to a subset of project options and new template level options. Since it's not clear what options are going to be in the initial version (waiting on design for those areas), this is purposefully left open until we have a bit more clarity on those details.

I was pushing off that work until the model code has been merged, i kinda figured there'd be a lot of test updates / random things and wanted to keep the complexity of this pr as close to sql changes as possible.